apps/test/integration/dropletIntegrationTests.js

```diff
@@ -0,0 +1,61 @@
+import {assert, expect} from '../util/configuredChai';
+import * as dropletUtils from '@cdo/apps/dropletUtils';
+import {singleton as studioApp, stubStudioApp, restoreStudioApp} from '@cdo/apps/StudioApp';
+import {loadSource} from './levelTests';
+
+describe('setParamAtIndex', () => {
+  let editor, parser, plainTree, arrayTree;
+
+  before(function (done) {
+    // Load droplet sources.
+    loadSource('/base/lib/ace/src-noconflict/ace.js')
+    .then(function () { return loadSource('/base/lib/ace/src-noconflict/mode-javascript.js'); })
+    .then(function () { return loadSource('/base/lib/ace/src-noconflict/ext-language_tools.js'); })
+    .then(function () { return loadSource('/base/lib/droplet/droplet-full.js'); })
+    .then(function () { return loadSource('/base/lib/tooltipster/jquery.tooltipster.js'); })
+    .then(function () { return loadSource('/base/lib/phaser/phaser.js'); })
+    .then(function () {
+      assert(window.droplet, 'droplet in global namespace');
+      done();
+    });
+  });
+
+  beforeEach(stubStudioApp);
+  afterEach(restoreStudioApp);
+
+  beforeEach(() => {
+    // Mock-up an editor to access a JS parser
+    const dropletCodeTextbox = document.createElement('div');
+    dropletCodeTextbox.setAttribute('id', 'dropletCodeTextbox');
+    editor = new window.droplet.Editor(dropletCodeTextbox, {
+      mode: 'javascript',
+      palette: [],
+      showPaletteInTextMode: false,
+      showDropdownInPalette: true,
+      allowFloatingBlocks: false,
+      enablePaletteAtStart: false,
+      textModeAtStart: true,
```

Comment:
Would this still work if we cut out the boolean options?  Just in case it helps readability of the test.